server/models/User.js

```diff
@@ -52,6 +52,10 @@ module.exports = (sequelize, DataTypes) => {
       foreignKey: 'userId',
       as: 'recipe'
     });
+    User.hasOne(models.Comment, {
```

Comment:
Ideally a user should be able to submit more than one comment on the whole platform 🙂

Reply:
Very valid. I have updated this